drivers/ieee802154/Kconfig

```diff
@@ -51,4 +51,8 @@ config IEEE802154_UPIPE_DRV_NAME
 	default "IEEE802154_UPIPE"
 	depends on IEEE802154_UPIPE
 
+config NETWORKING_WITH_15_4
+	bool
+	default n
+
```

Comment:
we have config IEEE802154, so why do we need this?

Reply:
see patch "include: net: net_pkt: Use NETWORKING_WITH_15_4 "